fdbserver/DataDistribution.actor.cpp

```diff
@@ -985,8 +1023,18 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 		state std::set<std::vector<UID>>::iterator teamIterEnd = self->primary ? initTeams->primaryTeams.end() : initTeams->remoteTeams.end();
 		for(; teamIter != teamIterEnd; ++teamIter) {
 			self->addTeam(teamIter->begin(), teamIter->end(), true);
+//			if ( !self->redundantTeamRemover.isReady() ) {
+//				wait( self->redundantTeamRemover );
+//			}
+			if ( self->redundantTeamRemover.isReady() ) {
+				self->redundantTeamRemover = teamRemover(self);
+				self->addActor.send(self->redundantTeamRemover);
+			}
+			self->traceTeamCollectionInfo();
 			wait( yield() );
 		}
+		// Start periodic team remover in the background
+		self->redundantTeamRemoverPeriodic = teamRemoverPeriodic(self);
```

Comment:
Should NOT have periodic teamRemover actor in the background. Once we add more team, we should call the teamRemover(). The previous simulation error is because we didn't call teamRemoved in the addSubsetTeamEmergency()